app.js

```diff
@@ -14,20 +15,26 @@ client.on('message', message => {
     message.channel.send(nbaUtil.getDaysOfWeek(nbaUtil.getCurrentMonday()))
   }
 
-	if (message.content === 'days') {
-		message.channel.send(nbaUtil.getDaysInWeek(nbaUtil.getCurrentMonday()));
-	}
-
-	if (message.content === 'player') {
-		let firstName = "Quinn";
-		let lastName = "Cook";
-		nbaData.getPlayers().then(
-			resp =>{
-				let teamId = nbaUtil.getTeamId(resp, firstName, lastName);
-				nbaData.getTeamSchedule(teamId).then(resp => console.log(resp));
-			}
-		);
-	}
+  if (message.content === 'player') {
+    let firstName = 'Quinn'
+    let lastName = 'Cook'
+    nbaData.getPlayers().then(
+      resp => {
+        let teamId = nbaUtil.getTeamId(resp, firstName, lastName)
+        nbaData.getTeamSchedule(teamId).then(resp => console.log(resp))
+      }
+    )
+  }
+  if (message.author.username !== client.user.username) {
+    if (message.channel.type === 'dm') {
+      const msg = message.content.split(' ')
+      if (msg[0] === 'setRefresh' && msg.length > 1) {
```

Comment:
Would there ever be a case where we would want the message length to be greater than 2? If not, we should guard against that. 
In either case though, we should also send back an error message if the input is incorrect.

Reply:
I fix